pkg/cli/cli.go

```diff
@@ -113,10 +168,28 @@ func (c cli) Run() error {
 	return c.cmd.Execute()
 }
 
-// WithCommandName is an Option that sets the cli's root command name.
-func WithCommandName(name string) Option {
+// WithRootCommandConfig configures a CLI's root command.
```

Comment:
`CLI` is the interface, `Option`s are defined on `cli`, not on `CLI`.
```suggestion
// WithRootCommandConfig configures a cli's root command.
```